internal/api/rust/rust.go

```diff
@@ -64,7 +64,10 @@ type RustClient struct {
 func NewRustClient(t ct.TestLike, opts api.ClientCreationOpts) (api.Client, error) {
 	t.Logf("NewRustClient[%s][%s] creating...", opts.UserID, opts.DeviceID)
 	matrix_sdk_ffi.LogEvent("rust.go", &zero, matrix_sdk_ffi.LogLevelInfo, t.Name(), fmt.Sprintf("NewRustClient[%s][%s] creating...", opts.UserID, opts.DeviceID))
-	ab := matrix_sdk_ffi.NewClientBuilder().HomeserverUrl(opts.BaseURL).SlidingSyncProxy(&opts.SlidingSyncURL).AutoEnableCrossSigning(true)
+	ab := matrix_sdk_ffi.NewClientBuilder()
+		.HomeserverUrl(opts.BaseURL)
+		.SlidingSyncVersionBuilder(&matrix_sdk_ffi.SlidingSyncVersionBuilderProxy{ Url: &opts.SlidingSyncURL })
```

Comment:
`SlidingSyncVersionBuilder` is supposed to receive an enum. I'm not sure it's the idiomatic way to do this in Go, but according to auto-generated code by `uniffi-go`, it seems to be alright.